index.html

```diff
@@ -4,32 +4,40 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="stylesheets.css">
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
     <title>Home Page</title>
 </head>
 
 <body>
-    <nav>
-        <a href="about.html">Info | </a>
-        <a href="pictures.html">Pictures</a>
-    </nav>
+    <div class="margin-left">
+        <nav>
+            <a href="about.html">Info | </a>
+            <a href="pictures.html">Pictures</a>
+        </nav>
```

Comment:
Let's have one <nav> element on this page.  Implement the following:
- Remove the nav nested in the <div class="margin-left">
- Move <div class="margin-left"> inside the remaining NAV element.

Reply:
hmm...sorry i am just now seeing this. We were gone for a few days. So, it appears to be glitching maybe and not all of the words that you typed have showed up, would you mind resending that comment?

Reply:
> hmm...sorry i am just now seeing this. We were gone for a few days. So, it appears to be glitching maybe and not all of the words that you typed have showed up, would you mind resending that comment?

Oh I see what you mean! Sorry about that -  there are 2 nav elements on index.html. Please remove one of them, so you'll only have one nav element remaining. 
2. Remove  the div wrapping everything in the body :` <div class="margin-left">`
3. In stylesheets.css apply the left margin directly to the body tag like this 
4. `body {
    margin-left: 30px
}`